django/db/models/sql/compiler.py

```diff
@@ -454,7 +454,13 @@ def as_sql(self, with_limits=True, with_col_aliases=False):
                 from_, f_params = self.get_from_clause()
                 where, w_params = self.compile(self.where) if self.where is not None else ("", [])
                 having, h_params = self.compile(self.having) if self.having is not None else ("", [])
-                result = ['SELECT']
+                result = []
+                if self.query.explain_query:
+                    prefix = self.connection.ops.explain_query_prefix(
+                        self.query.explain_format,
+                        self.query.explain_verbose)
```

Comment:
Hanging indent please, and it should be possible to remove the temporary `prefix`.

Reply:
Done 👍